ThunderCloud/EditLocalisationTableViewCell.swift

```diff
@@ -52,7 +52,7 @@ class EditLocalisationTableViewCell: InputTextViewCell {
 		super.layoutSubviews()
 		
 		backgroundView?.frame = CGRect(x: 8, y: 8, width: contentView.bounds.width-16, height: contentView.bounds.height - 8)
-		guard let cellTextLabel = cellTextLabel else {
+		guard let cellTextLabel = cellTextLabel?.superview else {
```

Comment:
It is yeah, because the label itself is in a container view which is the one which should have a RHS separator as defined in the lines below! Will rename the `let` now :)